worldedit-bukkit/src/main/resources/plugin.yml

```diff
@@ -1,9 +1,10 @@
-name: WorldEdit
+name: FastAsyncWorldEdit
 main: com.sk89q.worldedit.bukkit.WorldEditPlugin
 version: "${internalVersion}"
 load: STARTUP
 api-version: 1.13
```

Comment:
The `api-version` should adjusted to 1.15 here, as it doesn't support versions prior to.